src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -976,6 +976,69 @@ def test_aks_create_none_private_dns_zone(self, resource_group, resource_group_l
         self.cmd(
             'aks delete -g {resource_group} -n {name} --yes --no-wait', checks=[self.is_empty()])
 
+    # add live_only for role assignment GUID
```

Comment:
You can mock `_gen_guid` in role assignment: https://github.com/Azure/azure-cli-extensions/blob/fea5dcc42918ec56de7477854508b78e83c0da61/src/blueprint/azext_blueprint/tests/latest/test_blueprint_scenario.py#L93-L96

Reply:
wow, thank for the info! Changed to used mock.